string[] handness
float32[] normalized_hand_landmarks_left
float32[] normalized_hand_landmarks_right

